Extensions/ArtifactEngine/Providers/filesystemProvider.ts

```diff
@@ -68,14 +78,23 @@ export class FilesystemProvider implements models.IArtifactProvider {
             fs.readdir(itemsPath, (error, files) => {
```

Comment:
no need for try/catch around readdir?

Reply:
it returns errors is callback which is handled